server/commands/check_utils.go

```diff
@@ -426,7 +426,6 @@ func (t *tupleKeyIterator) next() (*openfgapb.TupleKey, error) {
 }
 
 func (t *tupleKeyIterator) stop() {
-	// Close shouldn't error if called multiple times
```

Comment:
?

Reply:
The function name in the comment was wrong, and it is repeated on the underlying `Stop` function. I don't think it is needed here. I could fix it and put it back though. WDYT?